File: Python/Data_Structures/Nodes/Node.py
class Node:
    def __init__(self, value, next_node=None):
        self.value = value
        self.next_node = next_node

    def get_value(self):
        return self.value

    def get_next(self):
        return self.next_node

    def set_value(self, value):
        self.value = value

    def set_next_node(self, node):
        self.next_node = node

File: C/Data_Structures/Nodes/node.c
#include "node.h"

int get_value(struct Node *node)
{
    return node->value;
}

struct Node *get_next_node(struct Node *node)
{
    return node->next_node;
}

void set_value(struct Node *node, int value)
{
    node->value = value;
}

void set_next_node(struct Node *node, struct Node *next)
{
    node->next_node = next;
}